node_modules/oae-content/lib/internal/util.js

```diff
@@ -104,7 +108,45 @@ var augmentContent = module.exports.augmentContent = function(ctx, content, dura
             content.previews.wideUrl = getSignedDownloadUrl(ctx, content.previews.wideUri, duration, offset);
             delete content.previews.wideUri;
         }
+
+    }
+};
+
+/**
+ * Generate a signed url that can be used to show video previews in the browser for a video file stored in AWS S3
+ *
+ * @param  {Context}    ctx               Current request context
+ * @param  {String}     content           The content object that needs a signed S3 video URL
+ * @param  {Function}   callback          Standard callback function
+ * @param  {Object}     callback.err      An error that occurred, if any
+ */
+var getSignedVideoUrl = module.exports.getSignedVideoUrl = function(ctx, content, callback) {
```

Comment:
Since the intention of this method is not to get something, but to add it to the content object, maybe we could name it `applySignedVideoUrl`. Alternatively, if this method just took in a ctx and video key and returned the video URL (the caller would be responsible for what to do with it -- assert it in a test, apply it to the content object, etc...) then the function name could stay the same.